src/createServer.js

```diff
@@ -1,3 +1,68 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable max-len */
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const bodyError = {};
+
+    bodyError.errors = [];
+
+    const [originalText, query] = req.url.slice(1).split('?');
+    const params = new URLSearchParams(query);
+    const targetCase = params.get('toCase');
+
+    if (!originalText) {
+      bodyError.errors.push({
+        message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    if (!targetCase) {
+      bodyError.errors.push({
+        message: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    if (targetCase) {
+      try {
+        convertToCase(originalText, targetCase);
+      } catch (err) {
+        bodyError.errors.push({
+          message: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+        });
+      }
+    }
```

Comment:
Create a separate helper that will get all errors, and keep error messages in errors constant